### DISCUSSION TITLE
Community meeting Jan 19, 2021

### DISCUSSION THREAD

**Discussion by carlisia** [vmware-tanzu/velero#3303] [Community meetings]
# Jan 19, 2021
### Status Updates
- carlisia
    - Almost done with addressing reviews for Download Request (https://github.com/vmware-tanzu/velero/pull/3004)

- bridget
    - Released v1.5.3
    - Helped out with some internal build processes
    - Working on POC for [multiple credentials](https://github.com/vmware-tanzu/velero/commit/8e967225319753f59f2e19c6cfa7da9fa1375cf8)
    - Need to review outstanding PRs
    
- nrb
    - Internal build updates for Velero v1.5.x series (build changes taking a while)
    - Reviewed timeout design w/ Phuong
        - My notes: https://hackmd.io/rGQuMa3XRDm7BW-CVVLAsg
- dave 
    - E2E tests running in AWS, Azure clouds
    - Working on next tests, CI/CD integration

### Discussion Topics
- carlisia
    - We are using GH Discussions for community support now (and other topics): https://github.com/vmware-tanzu/velero/discussions
- bridget
    - Multiple credentials POC demo
        - Velero PR: https://github.com/vmware-tanzu/velero/pull/3299
        - AWS plugin PR: https://github.com/vmware-tanzu/velero-plugin-for-aws/pull/69
- phuong: quick update on the plugin timeout design
    - Step 1. Add golang contexts into the plugin interfaces
        - Uses new `WithContext` variations to each fuction for backwards compatibility
    - Step 2. Determine timeouts & cancellation behavior
    - Plugin versioning: https://github.com/vmware-tanzu/velero/issues/3285
        - Helps with modifying plugin interfaces without requiring everyone upgrading in lock step
    - High level: looking for a pod-level timeout that covers ALL PVs connected.
    - Nolan's notes: https://hackmd.io/rGQuMa3XRDm7BW-CVVLAsg
    - Recording of the meeting: https://Dell.zoom.us/rec/share/EtlVa0vDa_fM0pzxArc9Izbq1LJwNpc1v5bfWcIwjCGzC4KDkjBqkudCWAGZqqI.e01a98H7-RO0y2_q Passcode: xH.vY3D5 
- Nolan & Dave will be doing a presentation on Velero at the Data Protection Working Group on Wednesday 1/26, 9am Pacific time
    - #wg-data-protection on the kubernetes slack

**Comment by jonasrosland** [comment]
YouTube recording: https://youtu.be/zSuunXz-_3c
